packages/cli/package.json

```diff
@@ -47,18 +47,18 @@
   },
   "dependencies": {
     "autoprefixer": "^8.4.1",
-    "babel-loader": "^7.0.0",
+    "@babel/plugin-syntax-dynamic-import": "^7.0.0-beta.47",
+    "@babel/plugin-transform-object-assign": "^7.0.0-beta.47",
+    "@babel/plugin-proposal-class-properties": "^7.0.0-beta.47",
+    "@babel/plugin-proposal-object-rest-spread": "^7.0.0-beta.47",
+    "@babel/plugin-transform-react-constant-elements": "^7.0.0-beta.47",
+    "@babel/plugin-transform-react-jsx": "^7.0.0-beta.47",
+    "@babel/preset-env": "^7.0.0-beta.47",
+    "babel-esm-plugin": "0.0.9",
+    "babel-loader": "^8.0.0-beta.3",
     "babel-plugin-jsx-pragmatic": "^1.0.2",
-    "babel-plugin-syntax-dynamic-import": "^6.18.0",
-    "babel-plugin-transform-class-properties": "^6.24.1",
-    "babel-plugin-transform-decorators-legacy": "^1.3.4",
     "babel-plugin-transform-export-extensions": "^6.22.0",
-    "babel-plugin-transform-object-assign": "^6.22.0",
-    "babel-plugin-transform-object-rest-spread": "^6.26.0",
-    "babel-plugin-transform-react-constant-elements": "^6.23.0",
-    "babel-plugin-transform-react-jsx": "^6.24.1",
     "babel-plugin-transform-react-remove-prop-types": "^0.4.5",
-    "babel-preset-env": "^1.3.3",
```

Comment:
Isn't this required? Preact build command fails at the moment

Reply:
look up a couple of lines, has to be changed in the babel config though

Reply:
https://github.com/developit/preact-cli/pull/583/files#diff-7234cda78ba01b8899055aae82c84a7fL7

Reply:
oh yeah...